strings/is_palindrome.py

```diff
@@ -0,0 +1,15 @@
+def is_palindrome(s):
+    """
+    Determine whether the string is palindrome
+    :param s:
+    :return: Boolean
+    """
+    return s == s[::-1]
+
+
+
+s = str(input("Enter string to determine whether its palindrome or not: ").strip())
```

Comment:
So the idea is that Travis will execute our script and since it does not have an interactive stdin stream connected to it, so an exception will be raised, for more info about the same please check the traceback in the failing tests, as mentioned by @cclauss 
You should only implement the algorithm and leave the invocation part of it. I would suggest to review the CONTRIBUTING.md once again :)